### PR TITLE
Add note about TypeScript debugging

### PR DESCRIPTION
**Pull request by vrubezhny**
Issue: #144

Signed-off-by: Victor Rubezhny <vrubezhny@redhat.com>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -4,10 +4,21 @@ This page describes the noteworthy improvements provided by each release of Ecli
 
 ### Next release...
 
+## 0.10.1
+
+* 📅 Release Date (tentative): Early September 2020
+* All changes: https://github.com/eclipse/wildwebdeveloper/compare/0.10.0...0.10.1
+
+#### TypeScript prorgam debug support
```
Comment (AObuchow) [inline RELEASE_NOTES.md:12]: Typo in “program”

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -4,10 +4,21 @@ This page describes the noteworthy improvements provided by each release of Ecli
...
+
+The TypeScript programs now can be directly debugged using `Node program` Debug Configuration.
```
Comment (AObuchow) [inline RELEASE_NOTES.md:14]: I would start the sentence with “TypeScript programs can now be directly debugged...” but this is subjective :)

File: RELEASE_NOTES.md (hunk 1 above)
Comment (vrubezhny) [reply RELEASE_NOTES.md:12]: yep

File: RELEASE_NOTES.md (hunk 2 above)
Comment (vrubezhny) [reply RELEASE_NOTES.md:14]: agreed

**Pull request merged**